Source/Plugin.BLE/Android/GattCallback.cs

```diff
@@ -174,7 +174,7 @@ public override void OnCharacteristicRead(BluetoothGatt gatt, BluetoothGattChara
 
             Trace.Message("OnCharacteristicRead: value {0}; status {1}", characteristic.GetValue().ToHexString(), status);
 
-            CharacteristicValueRead?.Invoke(this, new CharacteristicReadCallbackEventArgs(characteristic));
+            CharacteristicValueUpdated?.Invoke(this, new CharacteristicReadCallbackEventArgs(characteristic, status));
```

Comment:
Why do you change the invoked event to `CharacteristicValueUpdated` here? This leads to the warning:
GattCallback.cs(72,72): Warning CS0067: The event 'GattCallback.CharacteristicValueRead' is never used (CS0067)

Reply:
you are right. Copy and paste error. My bad.